src/main/java/frc/robot/Robot.java

```diff
@@ -182,10 +193,11 @@ public void teleopInit() {
     //SmartDashboard.putData(m_robotContainer.testFlup);
     if (m_autonomousCommand != null)
       m_autonomousCommand.cancel();
-    m_robotContainer.driveCommand.schedule();
+
+    //m_robotContainer.driveCommand.schedule();
```

Comment:
These should be undone. We want to match master/infiniteSimulator as much as possible

Reply:
It is done